scripts/json2db.pl

```diff
@@ -13,6 +13,14 @@
 
 my $parldata = mySociety::Config::get('RAWDATA');
 
+my $verbose = 0;
+for( @ARGV ){
+  if( $_ eq "--verbose" ){
```

Comment:
Could spacing generally match the existing spacing in the same file, just for consistency :)

Reply:
Amended this spacing.